content/en/account_management/rbac/permissions.md

```diff
@@ -306,7 +285,56 @@ This permission can be granted or revoked from a role via [the Roles API][1].
 
 ### Log Data Access
 
-The following permissions can be granted to manage read access on subsets of log data:
+Grant the following permissions to manage read access on subsets of log data:
+
+* `logs_read_data`(Recommended) offers finer grained access control over log data thanks to restriction queries.
+* `logs_read_index_data` is the alternative approach to restrict data access to indexed log data on a per-index basis.
+
+Those two permissions are applied sequentially meaning that a user can be restricted to a subset of indexes and the restriction queries apply on top of it.
+
+**Example**: User A has access to index `audit` and index `errors` and is restricted to the query `service:api`.
+When looking in Log Explorer, this user only sees logs from the `service:api` into the `audit` and `errors` indexes.
+
+In addition, access to the Live Tail can be restricted with the `logs_live_tail` permission regardless of the data access restriction of the user.
+
+#### logs_read_data
+
+Read access to log data. If granted, other restrictions then apply such as `logs_read_index_data` or with [restriction query][4].
```

Comment:
"The `logs_read_data` permission is required to grant a role read access to log data. Additionally, you can limit the scope of the role's access by applying a [Restriction Query](/api/?lang=bash#roles-restriction-queries-for-logs)."

"A Restriction Query is a logs query that restricts which logs the `logs_read_data` permission grants read access to. For users whose roles have Restriction Queries, any log query they make only returns those log events that also match one of their Restriction Queries. This is true whether the user queries log events from any log-related feature, including the log explorer, Live Tail, rehydration, or a dashboard widget."

^^ It may also be worth reiterating the relationship between logs_read_index_data and logs_read_data here.